src/core/Table/hooks/useResizeColumns.tsx

```diff
@@ -0,0 +1,440 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/**
+ * Copied from react-table as useResizerColumns does not have ownerDocument check also
+ * made some changes to resize sibling/next column on resize.
```

Comment:
"some changes" is not very helpful.

I tried to do a side-by-side diff and it was not obvious what changed (TS+lint changes basically every line), so I would suggest adding more info here or even inline about everything you changed/added.

Reply:
Added changes list.